### ISSUE TITLE
nftables variable `chain` is overwritten by default value `INPUT` in jail.conf

### ISSUE THREAD

**Issue by holymonson** [fail2ban/fail2ban#1949]
### Environment:

_Fill out and check (`[x]`) the boxes which apply. If your Fail2Ban version is outdated, 
and you can't verify that the issue persists in the recent release, better seek support 
from the distribution you obtained Fail2Ban from_

- Fail2Ban version (including any possible distribution suffixes): 0.10.0-1
- OS, including release name/version: 4.13.10-1-ARCH
- [x] Fail2Ban installed via OS/distribution mechanisms
- [x] You have not applied any additional foreign patches to the codebase
- [ ] Some customizations were done to the configuration (provide details below is so)

### The issue:

When using action `nftables` instead of `iptables`, the variable `chain ` is overwritten incorrectly.

In `action.d/nftables-common.conf`:
```
# Option:  chain
# Notes    specifies the nftables chain to which the Fail2Ban rules should be
#          added
# Values:  STRING  Default: input
chain = input
```
And in `action.d/iptables-common.conf`:
```
# Option:  chain
# Notes    specifies the iptables chain to which the Fail2Ban rules should be
#          added
# Values:  STRING  Default: INPUT
chain = INPUT
```
#### Steps to reproduce
Use nftables to ban.

#### Any additional information
If setup a nftables chain `INPUT` in advance, fail2ban can correctly insert rules into that chain.

#### Any customizations done to /etc/fail2ban/ configuration
```
[DEFAULT]
bantime = 1d
banaction = nftables-multiport
banaction_allports = nftables-allports
```

#### Relevant parts of /var/log/fail2ban.log file:
_preferably obtained while running fail2ban with `loglevel = 4`_

```
2017-10-30 02:43:47,065 fail2ban.utils          [10733]: Level 39 ffffa9c1cc68 -- exec: nft add set inet filter f2b-sshd \{ type ipv4_addr\; \}
nft insert rule inet filter INPUT tcp dport \{ ssh \} ip saddr @f2b-sshd reject
2017-10-30 02:43:47,067 fail2ban.utils          [10733]: ERROR   ffffa9c1cc68 -- stderr: '<cmdline>:1:1-74: Error: Could not process rule: No such file or directory'
2017-10-30 02:43:47,068 fail2ban.utils          [10733]: ERROR   ffffa9c1cc68 -- stderr: 'insert rule inet filter INPUT tcp dport { ssh } ip saddr @f2b-sshd reject'
2017-10-30 02:43:47,069 fail2ban.utils          [10733]: ERROR   ffffa9c1cc68 -- stderr: '^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^'
2017-10-30 02:43:47,070 fail2ban.utils          [10733]: ERROR   ffffa9c1cc68 -- returned 1
2017-10-30 02:43:47,073 fail2ban.actions        [10733]: ERROR   Failed to start jail 'sshd' action 'nftables-multiport': Error starting action Jail('sshd')/nftables-multiport
```

#### Relevant lines from monitored log files in question:

```
```

**Comment by sebres** [comment]
You are wrong a bit with your conclusion: this is default value of [`chain` (set to `INPUT`)](https://github.com/fail2ban/fail2ban/blob/0.10/config/jail.conf#L148) in `jail.conf`, that overwrites it in interpolation below via definition in [action_](https://github.com/fail2ban/fail2ban/blob/0.10/config/jail.conf#L168).

Thus to use `nft` currently you should specify in your jail.local:
```INI
[DEFAULT]
banaction = nftables-multiport
banaction_allports = nftables-allports
chain = input
```

**Comment by sebres** [comment]
Thinking a bit more about this, I suspect that we can fix it for fail2ban >= v.0.10, by redefining a parameter `chain` in `jail.conf` to `<known/chain>` or `<default/chain>`...
Thus reopened.

**Comment by sebres** [comment]
Fixed in b615a9854019589c87d68dd586f45c61f8fe886d
